src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallServiceTest.kt

```diff
@@ -54,11 +55,12 @@ class RecallServiceTest {
   private val bankHolidayService = mockk<BankHolidayService>()
   private val prisonerOffenderSearchClient = mockk<PrisonerOffenderSearchClient>()
   private val prisonApiClient = mockk<PrisonApiClient>()
-  private val meterRegistry = mockk<MeterRegistry>()
+  private val meterRegistry = mockk<MeterRegistry>(relaxed = true)
+  private val autoReturnedToCustodyCounter = mockk<Counter>()
   private val fixedClock = Clock.fixed(Instant.parse("2021-10-04T13:15:50.00Z"), ZoneId.of("UTC"))
   private val returnToCustodyUpdateThresholdMinutes = 60L
-  private val underTest =
-    RecallService(recallRepository, bankHolidayService, prisonerOffenderSearchClient, prisonApiClient, fixedClock, meterRegistry, returnToCustodyUpdateThresholdMinutes)
+
+  private var underTest = mockk<RecallService>()
```

Comment:
not sure why we're setting it as a `mock<RecallService>()`. It def shouldn't ever be a mock

Reply:
I tried `null` but I couldn't then re-assign it as a `RecallService` in the `@BeforeEach` block...

I could just assign it as before here and then re-assign after the two `every` blocks in the `@BeforeEach`, happy to do whatever...

Reply:
`private lateinit var underTest: RecallService`